src/utils/string.ts

```diff
@@ -13,11 +13,18 @@ export function trimForwardSlash(str: string): string {
  *
  * @param str - The string to process.
  * @param prefix - The prefix to remove.
+ * @param [limit] - Maximum amount of times to trim. No limit by default.
```

Comment:
(nit) weird default behaviour imo

Reply:
Can be an f/u

Reply:
https://github.com/SkynetHQ/skynet-js/issues/27